tests/integration/test_requirements_pyyaml.py

```
@@ -0,0 +1,241 @@
+"""
+Comprehensive tests for PyYAML dependencies in requirements-dev.txt.
+Tests the addition of PyYAML and types-PyYAML dependencies.
+"""
+
+import pytest
+import re
+from pathlib import Path
+from typing import List
```

Ruff (0.14.6)

`typing.List` is deprecated, use `list` instead

(UP035)